addons/web/static/tests/webclient/actions/window_action_tests.js

```diff
@@ -2286,8 +2286,12 @@ QUnit.module("ActionManager", (hooks) => {
         ]);
     });
 
-    QUnit.test(
-        "doAction supports being passed controlletState (searchModel)",
+    QUnit.skip(
+        "doAction supports being passed controllerState (searchModel)",
+        /** with the dashboard view refactoring, the prop searchModel has become
+         * useless under the present form.
```

Comment:
you can still pass the sharedState, can't you?

Reply:
see comment in code